### PR TITLE
DiegoCardenas_HW01

### PR DESCRIPTION
**Pull request by diegocardenas1**
I got the changes into README.md basically by forking the HW01 assignment from uc-cfss and then cloning that into RStudio based on the instructions on the website (i.e., starting a new project and using the URL of the forked HW01 file).  When I was trying to create the biography and knit/preview the code, I kept getting an error message that said something along the lines of "pandoc document conversion failed with error 2." I googled the error and the github support section said that I hadn't installed a particular devtool with "rstudio/rmarkdown." Perhaps this information was on the course website and I just couldn't find it, but I lost whatever tools we installed in class first week because I've been working in Crerar and waiting for my personal laptop to get mailed back to me in a fixed condition. I was also having some problems with the Markdown syntax, so I went to the www.markdowntutorial.com/ and practiced writing some code. I found out I had the spacing wrong on the headers. Overall, using Markdown was fun and relatively easy. My Github workflow was generally an easy thing to navigate as well, though finding exactly where to create a pull request was kind of difficult at first. I just had to read the instructions on the site. Overall, this was a bit of a learning experience, but I liked using Markdown to make a biography!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,23 @@
-# hw01
-Repository for homework 1
+# A Biography of Diego Cardenas
+
+### Basic Information
+I am a **fourth-year sociology major** in the college, and I was born and raised in **Phoenix, Arizona.** I want to go to graduate school some day to pursue a PhD in **Sociology**, but I plan on taking a few gap years for now. I'm not a big fan of extremely hot and dry weather, but it's not like I like Chicago's winters either.
+
+### How I look
+
+![picture of me](Diego.png)
```
Comment (gmvelez) [inline README.md:8]: This is a good strategy to save images in the repo and then call them this way.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,23 @@
...
+
+Some of my interests include the following:
+
+* Playing and watching basketball.
+* Conducting sociological research.
+* Reading about urban sociology and ethnography.
+* Listening to music.
+* Hiking.
+* Doing community service.
+
```
Comment (gmvelez) [inline README.md:20]: Nice details and use of other formatting in R Markdown

**Review by gmvelez — COMMENTED** [review]
Evaluation HW-01 for Diego Cardenas
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |           |      x        |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |      x     |              |            |

Good job writing a complete bio, using multiple formatting techniques, and successfully linking a picture from your repo.  Your reflection was also thorough.

**Comment by e-aoki** [comment]
Coding Style - Excellent
Coding Strategy - Excellent
Presentation - Excellent
Achievement, Creativity - Excellent
Ease of Access - Excellent

Great job! I thought that constantly pushing commits and saving them was a great strategy moving forward. Even though this was a smaller assignment, it willl definitely be useful in the future!

**Comment by lmulholland42** [comment]
Review of HW01 for Diego Cardenas

Coding style: Excellent
Coding strategy: Excellent
Presentation: Excellent
Achievement, creativity: Excellent
Ease of access: Excellent

Remarks:
This was really well formatted and consistent. I like that you used a lot of the different formatting styles as well. I don't have any critiques.

**Pull request closed without merging**